docs/retrieve.txt

```diff
@@ -0,0 +1,342 @@
+is ``"The Parent Trap"``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: php
+      
+      return view('browse_movies', [
+         'movies' => Movie::where('title', 'The Parent Trap')
+            ->first()
+      ]);
+
+   .. output::
+      :language: none
+      :visible: false
+
+      Title: The Parent Trap
+      Year: 1961
+      Runtime: 129
+      IMDB Rating: 7.1
+      IMDB Votes: 12768
+      Plot: Teenage twin girls swap places and scheme to reunite their divorced parents.
+
+Without chaining the ``first()`` method to the ``where()`` method, the preceding example returns
+two documents in which the ``title`` value is ``"The Parent Trap"``.
+
+.. important:: Required File Changes
+
+   To run the preceding code example, you must edit the ``MovieController.php`` file in your
+   ``my-app`` application. Copy the code example and paste it into the ``show()`` function, replacing
+   any existing code inside this function.
+
+Use Laravel Queries to Retrieve Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Rather than on a model, you can run find operations on a query builder instance that
+represents your collection. To construct a query builder instance, import the ``DB``
+facade into your controller file and use Laravel's database query builder syntax.
+
+The following example uses Laravel's query builder to retrieve documents in which the value
+of the ``imdb.votes`` nested field is greater than ``1300000``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: php
+
+      return view('browse_movies', [
+         'movies' => DB::collection('movies')
+               ->where('imdb.votes', '>', 1300000)
+               ->get()
+      ]);
+
+   .. output::
+      :language: none
+      :visible: false
+
+      Title: The Shawshank Redemption
+      Year: 1994
+      Runtime: 142
+      IMDB Rating: 9.3
+      IMDB Votes: 1513145
+      Plot: Two imprisoned men bond over a number of years, finding solace and eventual redemption
+      through acts of common decency.
+
+      Title: The Dark Knight
+      Year: 2008
+      Runtime: 152
+      IMDB Rating: 9
+      IMDB Votes: 1495351
+      Plot: When the menace known as the Joker wreaks havoc and chaos on the people of Gotham, the
+      caped crusader must come to terms with one of the greatest psychological tests of his ability to
+      fight injustice.
+
+.. important:: Required File Changes
+
+   To run the preceding code example, you must make the following changes to your ``my-app``
+   application:
+
+   - Copy the code example and paste it into the ``show()`` function in your ``MovieController.php`` file
+   - Add the ``use Illuminate\Support\Facades\DB`` use statement to your ``MovieController.php``
+     file
+   - Replace ``DB_CONNECTION=mysql`` with ``DB_CONNECTION=mongodb`` in your ``.env`` file
+   - Replace the contents of your ``browse_movies.blade.php`` file with the following code:
```

Comment:
There's a small difference: the DB:collection query returns an array rather than an object (I believe), so you can't use the `->` object operator and have to use brackets instead

Reply:
Ah, thanks for answering, that is good to know. Since this is a difference between using Eloquent models and the facade, could that be included in the explanation of the latter? I think including it could potentially help readers that might not expect that difference.

Reply:
Ticketed this update to the Quick Start [here](https://jira.mongodb.org/browse/DOCSP-36792).